ambry-store/src/main/java/com.github.ambry.store/LogSegment.java

```diff
@@ -180,6 +182,37 @@ public void appendFrom(ReadableByteChannel channel, long size) throws IOExceptio
     }
   }
 
+  /**
+   * {@inheritDoc}
```

Comment:
same here - not inheriting from anything (as opposed to the other two `append..()` functions

Reply:
Fixed.